server/app/controllers/admin/AdminProgramBlockQuestionsController.java

```diff
@@ -20,6 +20,7 @@
 import services.program.CantAddQuestionToBlockException;
 import services.program.IllegalPredicateOrderingException;
 import services.program.InvalidQuestionPositionException;
+import services.program.ProgramBlockDefinitionContainsInvalidQuestionException;
```

Comment:
This is a verrrrrry long name.  Open to suggestions to make this more concise.  Could be used in the future for other invalid questions in a program block.

Reply:
It is informative, but could be `ProgramQuestionDefinitionInvalidException`.

Reply:
I like that better 👍